BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.m

```diff
@@ -323,18 +312,10 @@ - (void)executeOperation
 
         if (self.error == nil && ![self isCancelled])
         {
-            //FIXME: remove shouldUseSessionTask and this condition after switching
-            //completely from NSURLConnection to NSURLSessionTask
-            if (self.shouldUseSessionTask == YES) {
-                if (self.sessionTask == nil) {
-                    self.sessionTask = [self createSessionTask];
-                }
-                [self.sessionTask resume];
-            } else {
-                self.connection = [[NSURLConnection alloc] initWithRequest:self.APIRequest delegate:self];
-                BOXLog(@"Starting %@", self);
-                [self startURLConnection];
+            if (self.sessionTask == nil) {
```

Comment:
Minor: We should use _sessionTask here when we're checking == nil. `createSessionTask` could be renamed `sessionTask`.